torch/csrc/autograd/python_function.cpp

```diff
@@ -1264,41 +1315,6 @@ int THPFunction_set_materialize_non_diff_grads(
   END_HANDLE_TH_ERRORS_RET(-1)
 }
 
-static PyObject* unpack_saved_variables(
```

Comment:
Is this code just moved?  Why?  Did anything change?

Reply:
it was moved since code above it needed to unpack, I could of instead forward declared it. move is no longer needed